frontend/packages/integration-tests-cypress/views/overview.ts

```diff
@@ -0,0 +1,14 @@
+export const overviewPage = {
+  projectOverviewIsVisible: () => {
+    cy.get(`.odc-topology-list-view`).should('be.visible');
+  },
+  itemsAreVisible: () => {
+    cy.get(`.odc-topology-list-view__item-row`).should('be.visible');
+  },
+  groupLabelItemPresent: (kind) => {
+    cy.get('.odc-topology-list-view__kind-label').contains(kind.label);
+  },
+  projectOverviewListItemPresent: (name) => {
```

Comment:
`groupLabelItemContains`
`projectOverviewListItemContains`

Also, how do you feel about #8878?  I updated for consistency and readability but I think it's somewhat redundant to have `.contains('..').should('exist')`

Reply:
@dtaylor113 done; and agreed `.contains()` followed by `should('exist')` seems redundant.